statestores/mongo/statestore.go

```diff
@@ -31,16 +31,16 @@ type StateStore struct {
 }
 
 type StateStoreSettings struct {
-	ConnectionString string
+	ConnectionString     string
 	ConnectionStringType connectorMongo.MongoFlavor
 	serverConnectTimeout time.Duration
 	pingTimeout          time.Duration
 }
 
 func NewMongoStateStore(settings StateStoreSettings) *StateStore {
 	settings.serverConnectTimeout = 20 * time.Second
-	settings.pingTimeout = 2 * time.Second
-	settings.ConnectionStringType = connectorMongo.GetMongoFlavor(settings.ConnectionString);
+	settings.pingTimeout = 10 * time.Second
```

Comment:
ahhhhh my arbitrary timeouts that are inconsistent with the settings file

Reply:
What is technical debt? ;)